src/gui/widgets/GroupBox.cpp

```diff
@@ -99,35 +99,18 @@ void GroupBox::updatePixmap()
 	QColor bg_color = QApplication::palette().color( QPalette::Active,
```

Comment:
Then, I guess I'll also have to make the outer rectangle's color themeable also?

Or, would "p.background().color().darker( 150 )" sufficient?